ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -25,57 +26,86 @@ import {
 } from 'testing/stubs/institution-api-stub';
 
 import { AdminInstitutionEdit } from './admin-institution-edit';
+import { MembershipRequirements } from './admin-institution-options';
+
+const getCTAddress = () => screen.getByTestId('controlled-email-address');
+const getCTAddressInput = (): HTMLInputElement =>
+  screen.getByTestId('controlled-email-address-input');
+const getCTDetails = () => screen.getByTestId('controlled-card-details');
+const getCTDomain = () => screen.getByTestId('controlled-email-domain');
+const getCTDomainInput = (): HTMLInputElement =>
+  screen.getByTestId('controlled-email-domain-input');
+const getCTDropdown = () => screen.getByTestId('controlled-agreement-dropdown');
+const getCTEnabled = (): HTMLInputElement =>
+  screen.getByTestId('controlled-enabled-switch');
+
+const getRTAddress = () => screen.getByTestId('registered-email-address');
+const getRTAddressInput = (): HTMLInputElement =>
+  screen.getByTestId('registered-email-address-input');
+const getRTDetails = () => screen.getByTestId('registered-card-details');
+const getRTDomain = () => screen.getByTestId('registered-email-domain');
+const getRTDomainInput = (): HTMLInputElement =>
+  screen.getByTestId('registered-email-domain-input');
+const getRTDropdown = () => screen.getByTestId('registered-agreement-dropdown');
+
+const queryCTAddress = () => screen.queryByTestId('controlled-email-address');
+const queryCTAddressInput = () =>
+  screen.queryByTestId('controlled-email-address-input');
+const queryCTDetails = () => screen.queryByTestId('controlled-card-details');
+const queryCTDomain = () => screen.queryByTestId('controlled-email-domain');
+const queryCTDomainInput = (): HTMLInputElement =>
+  screen.queryByTestId('controlled-email-domain-input');
+
+const queryRTAddress = () => screen.queryByTestId('registered-email-address');
+const queryRTAddressInput = (): HTMLInputElement =>
+  screen.queryByTestId('registered-email-address-input');
+const queryRTDomain = () => screen.queryByTestId('registered-email-domain');
+const queryRTDomainInput = (): HTMLInputElement =>
+  screen.queryByTestId('registered-email-domain-input');
```

Comment:
These don't match 1 for 1 the old queries, because I changed the prefix to get and query instead of the pre-existing find so as not to be confused with RTL's find functionality.